docs-ui/components/colors.stories.js

```diff
@@ -2,32 +2,72 @@ import React from 'react';
 import {withInfo} from '@storybook/addon-info';
 import styled from '@emotion/styled';
 
-import theme from 'app/utils/theme';
+import theme, {aliases} from 'app/utils/theme';
 
 export default {
   title: 'Style/Colors',
 };
 
+const DESCRIPTIONS = {
+  textColor: 'Primary text color',
+  subText: 'Text that should not have as much emphasis',
+  bodyBackground: 'Background for the main content area of a page?',
+  background: 'Primary background color',
+  backgroundSecondary:
+    'Secondary background color used as a slight contrast against primary background',
+  headerBackground: 'Background for the header of a page',
+  border: 'Primary border color',
+  success: 'A color that denotes a "success", or something good',
+  error: 'A color that denotes an error, or something that is wrong',
+  disabled:
+    'A color that indicates something is disabled where user can not interact or use it in the usual manner (implies that there is an "enabled" state)',
+  active: 'Indicates that something is "active" or "selected"',
+  linkColor: 'Link color indicates that something is clickable',
```

Comment:
Hmm.. should we really add a description for every color? In my opinion, the color name should be self-descriptive. For example, instead of `linkColor` I would write `clickableLinkColor`  or `linkColorClickable`

Reply:
Sure, but does it hurt to have both? It's additional guidance on when/where you should use it

Reply:
yeah it doesn't hurt 😄